flutter_kernel_transformers/pubspec.lock

```diff
@@ -0,0 +1,335 @@
+# Generated by pub
```

Comment:
this is a library, right? it shouldn't need a pubspec.lock

Reply:
it is a package same as frontend_server. That was the design @aam aam recommended. I don't have a strong preference once way or the other.

Reply:
removed pubspec.lock file. it shouldn't be in the CL.

Reply:
I believe the difference between frontend_server and flutter_kernel_transformers is that the former is an app (it's run directly) whereas the latter is a library (it's imported by the other). Is that an incorrect interpretation?